versions.yml

```diff
@@ -1,4 +1,4 @@
-fabricVersion: 0.14.22
+fabricVersion: 0.14.23
```

Comment:
No, I don't think this is needed. This doesn't fix anything, only for 1.20.3. And updating fabric versions is bit wonky. So unless there is a reason for this, I want this reverted

Reply:
I always updated fabric with my updates and never found any issues. The only problem is when you have Minecraft Launcher open when updating but all you have to do then is just restart the launcher.
But if you wish I can revert it when I came back home.

Reply:
I personally had to delete my instance even when MC was closed. Also seen some reports like that. So if not necessary, I wouldn't do it.